tools/bridge/bridge/main.py

```diff
@@ -234,6 +235,40 @@ def make_validator_balance_watcher(config, control_queue):
     )
 
 
+public_config_keys = (
```

Comment:
Why can't all config keys be public?

Reply:
Some of them contain confidential information, like the validators private key.